docs/examples/MarrmotM14.ipynb

```diff
@@ -2,204 +2,157 @@
  "cells": [
```

Comment:
<p>Isnt end_time also a model parameter? Can this sentence be rephrased to something like</p><p> </p><ul><li>Setup model with maximum soil moisture storage of 12 instead of the default 1000 and an earlier end time making total model time just 1 month.</li></ul><p>or</p><ul><li>Adjust model parameters using <code>setup()</code> function</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
it is revised as you suggested.